src/integration-test/java/com/commercetools/project/sync/CliRunnerIT.java

```diff
@@ -417,23 +474,26 @@ void run_WithSyncAsArgumentWithAllArgAsFullSync_ShouldExecuteAllSyncers() {
     // create clients again (for assertions and cleanup), since the run method closes the clients
     // after execution
     // is done.
-    try (final SphereClient postSourceClient = createClient(CTP_SOURCE_CLIENT_CONFIG)) {
-      try (final SphereClient postTargetClient = createClient(CTP_TARGET_CLIENT_CONFIG)) {
-        // assertions
-        assertAllSyncersLoggingEvents(syncerTestLogger, cliRunnerTestLogger, 1);
-
-        assertAllResourcesAreSyncedToTarget(postTargetClient);
-        assertCurrentCtpTimestampGeneratorDoesntExist(
-            postTargetClient, "runnerName", "ProductTypeSync");
-        assertNoCustomObjectExists(postTargetClient);
-      }
+    try (final SphereClient postTargetClient = createClient(CTP_TARGET_CLIENT_CONFIG)) {
+      // assertions
+      assertAllSyncersLoggingEvents(syncerTestLogger, cliRunnerTestLogger, 1);
+
+      assertAllResourcesAreSyncedToTarget(postTargetClient);
+      assertCurrentCtpTimestampGeneratorDoesntExist(
+          postTargetClient, "runnerName", "ProductTypeSync");
+      assertNoProjectSyncCustomObjectExists(postTargetClient);
     }
   }
 
-  private void assertNoCustomObjectExists(@Nonnull final SphereClient targetClient) {
-
+  private void assertNoProjectSyncCustomObjectExists(@Nonnull final SphereClient targetClient) {
     final CustomObjectQuery<LastSyncCustomObject> lastSyncQuery =
-        CustomObjectQuery.of(LastSyncCustomObject.class);
+        CustomObjectQuery.of(LastSyncCustomObject.class)
+            .plusPredicates(
+                customObjectQueryModel ->
+                    customObjectQueryModel
+                        .container()
+                        .is(
+                            "commercetools-project-sync.runnerName.ProductSync.timestampGenerator"));
```

Comment:
The string is used inside the test class twice. Can it defined as a private constant?

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/168/commits/8ec4da6225be99af9207f693da29254a691a3acd